docs/api/algorand/submit-transaction.mdx

```diff
@@ -0,0 +1,71 @@
+# SubmitTransaction
+
+Send funds from a [DelegatedWallet](#TODO), to another (regular or delegated) wallet.
```

Comment:
Please clarify whether the regular wallet will be opted in, linking `/docs/concepts/tokens#token-opt-in`

Reply:
Ah no sorry, this is a refuse from another page. Forgot to add a proper description